File: java/challenges/KilometerConverter.java

import java.util.Scanner;

public class KilometerConverter
{
  public static void main(String[] args) 
  {
    displayIntro();
    kilometerToMileConverter();
  }

  public static void displayIntro() 
  {
    System.out.println("Welcome to the Kilometer to Miles Converter!");
  }

  public static void kilometerToMileConverter() 
  {
    // Declare variables
    float kilometers;
    double miles;
    final double KILOMETER_TO_MILE_CONVERSION_RATE = 0.6214;
    Scanner keyboard = new Scanner(System.in);

    // Gathering input from user
    System.out.println("Please enter the amount in kilometer(s)"); 
    kilometers = keyboard.nextInt();

    // Calculating miles
    miles = kilometers * KILOMETER_TO_MILE_CONVERSION_RATE;

    // Displaying output
    System.out.println( kilometers + " kilometer(s) is equal to " + miles + " mile(s)."); 

    // Close keyboard
    keyboard.close();
  }
}

File: python/challenges/kilometer_converter.py
def main():
  display_intro()
  kilometer_to_mile_converter()

def display_intro():
  print("Welcome to the Kilometer to Miles Converter!\n")

def kilometer_to_mile_converter():
  # Declare constant
  KILOMETER_TO_MILE_CONVERSION_RATE = 0.6214

  # Gathering input from user
  kilometers = int(input("Please enter the amount in kilometer(s)\n"))
  
  # Calculating miles
  miles = kilometers * KILOMETER_TO_MILE_CONVERSION_RATE

  # Displaying output
  print(f"{kilometers} kilometer(s) is equal to {miles} mile(s).")

main()